doc/Building Tribler on Windows.md

```diff
@@ -0,0 +1,43 @@
+This page contains information about building Tribler on Windows. The final result you should have is a `.exe` file which, when opened, enables users to install Tribler on their system. 
+This guide installs a 64-bit version of Tribler and has been tested on Windows 10 and Windows 2008 Server R2, 64-bit. It is recommended to run this builder on a system that is already able to run Tribler without problems (it means that all the required packages required by Tribler are installed already).  In case you want to build a 32 bit version, just install all the dependencies mentioned here in 32 bit version.
+Information about setting up a developer environment on Windows can be found [here](https://github.com/Tribler/tribler/blob/devel/doc/Tribler%20development%20on%20Windows.md).
+
+**When you have installed zope, an empty ``__init__.py`` file must be present in the zope folder. If this file is missing, a ``No module named zope`` error will be thrown. Create this file in the ``site-packes/zope`` folder if it does not exist.**
+ 
+# Required packages
+To build a Tribler installer, you'll need some additional scripts and packages:
+* The git command tools are required to fetch the latest release information. These can be downloaded from [here](https://git-scm.com/download/win). 
```

Comment:
Could we have a list of versions that are known to be working for us? Just to save time to other people if there's any issue with the latest releases at that point in time.